notepad/Notepad.js

```diff
@@ -16,6 +16,33 @@ var ImageTool = require('substance/packages/image/ImageTool');
 var MarkTool = require('./MarkTool');
 var TodoTool = require('./TodoTool');
 
+Controller.prototype.uploadFile = function(file, cb) {
```

Comment:
Thx!
Also error handling probably not the best.

On Tue, Feb 16, 2016, 19:32 Michael Aufreiter <notifications@github.com>
wrote:

> In notepad/Notepad.js
> <https://github.com/substance/notepad/pull/19#discussion_r53025922>:
>
> > @@ -16,6 +16,33 @@ var ImageTool = require('substance/packages/image/ImageTool');
> >  var MarkTool = require('./MarkTool');
> >  var TodoTool = require('./TodoTool');
> >
> > +Controller.prototype.uploadFile = function(file, cb) {
>
> It's not good practice to monkey patch the Controller prototype. Instead
> you can just add that method to the Notepad prototype. I will do it in the
> course of my refactor anyway. Just to let you know for next time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/substance/notepad/pull/19/files#r53025922>.
>